scripts/defaultConfig.json

```diff
@@ -8,7 +8,8 @@
           "electrumRpcEndpoint": "http://swapper:swapper@electrum-regtest:30000",
           "bitcoincoreZmqEndpoint": "tcp://bitcoincore-regtest:28332",
           "bitcoinAddressCallbackHost": "http://connector:80",
-          "electrsEndpoint": "electrs-regtest:60401:t"
+          "electrsHost": "electrs-regtest",
```

Comment:
To be consistent with the other parameters, it should be only one. We can split the value inside the code